specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2019-08-01/notebook.json

```diff
@@ -0,0 +1,384 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "Cosmos DB",
+		"description": "Azure Cosmos DB Database Service Resource Provider REST API",
+		"version": "2019-08-01"
+	},
+	"host": "management.azure.com",
+	"schemes": [
+		"https"
+	],
+	"consumes": [
+		"application/json"
+	],
+	"produces": [
+		"application/json"
+	],
+	"security": [{
+		"azure_auth": [
+			"user_impersonation"
+		]
+	}],
+	"securityDefinitions": {
+		"azure_auth": {
+			"type": "oauth2",
+			"authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+			"flow": "implicit",
+			"description": "Azure Active Directory OAuth2 Flow",
+			"scopes": {
+				"user_impersonation": "Impersonate your user account"
+			}
+		}
+	},
+	"paths": {
+		"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/notebookWorkspaces": {
+			"get": {
+				"tags": [
+					"NotebookWorkspacesResource"
+				],
+				"description": "Gets the notebook workspace resources of an existing Cosmos DB account.",
+				"operationId": "NotebookWorkspaces_ListByDatabaseAccount",
+				"x-ms-pageable": {
+					"nextLinkName": null
+				},
+				"parameters": [{
+						"$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/subscriptionIdParameter"
+					},
+					{
+						"$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/resourceGroupNameParameter"
+					},
+					{
+						"$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/apiVersionParameter"
+					},
+					{
+						"$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/accountNameParameter"
+					}
+				],
+				"responses": {
+					"200": {
```

Comment:
For all API responses in this spec, please add a "default" response to capture error cases. It should have a schema consistent with the [common error contract](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-details.md#error-response-content). 
Here is an example:
https://github.com/Azure/azure-rest-api-specs/blob/0cf70260108cb40bc68d408674c187ed065f8c64/specification/batch/resource-manager/Microsoft.Batch/stable/2019-08-01/BatchManagement.json#L105